.azure/scripts/metadata_validator.py

```diff
@@ -14,7 +14,7 @@
 
 VALID_BUILD_PACKS = [
     "ballerina", "wso2-mi", "go", "java", "php", "python", "nodejs", "ruby",
-    "vue", "many", "postman", "react", "docker", "angular", "dotnet"
+    "vue", "many", "postman", "react", "docker", "angular", "dotnet", "prism-mock"
```

Comment:
Make sure to add the new buildpack to the choreo-console as well - https://github.com/wso2-enterprise/choreo-console/blob/d3327dd7258bea1fa6dbe019fe9bbf3b3dc5e46b/src/hooks/samples.tsx#L38

Reply:
+1. Let's hold this PR until we add the build pack details in Choreo Console.

Reply:
Ack.